src/frontend/src/flows/manage/authenticatorsSection.ts

```diff
@@ -96,7 +99,7 @@ export const authenticatorsSection = ({
         <div class="c-action-list">
           <ul>
           ${authenticators.map((authenticator, index) =>
-            authenticatorItem({ authenticator, index, i18n })
+            authenticatorItem({ authenticator, index, i18n, onRemoveDevice })
```

Comment:
As suggested earlier, from a UI perspective we only have one thing to remove within the `authenticatorItem` component, so the following should suffice.

```suggestion
            authenticatorItem({ authenticator, index, i18n, onRemove: () => onRemoveDevice(authenticator. device) })
```

Reply:
Sure